### PR TITLE
Cockpit 223

### PR DESCRIPTION
**Pull request by KKoukiou**
Page preview: https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-223.html

### CODE REVIEW

File: _posts/2020-07-08-cockpit-223.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+category: release
+summary: Cockpit with web server hardening
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 223.
+
+## Webserver: Standard-conformant lifetime of web server certificate
+
+Cockpit's web server creates a self-signed certificate `0-self-signed.cert` on startup if the administrator did not already install one. They now are valid for one year, following the [official standard](https://www.globalsign.com/en/blog/ssl-certificate-validity-capped-at-maximum-two-years). The previous lifetime of 100 years for a self-signed certificate or 10 years with sscg may not be accepted by some browsers in the future any more.
```
Comment (allisonkarlitskaya) [inline _posts/2020-07-08-cockpit-223.md:16]: Add parens or dashes around (or 10 years with sscg)

Drop the "any more".

Comment (martinpitt) [reply _posts/2020-07-08-cockpit-223.md:16]: Good point, thanks @allisonkarlitskaya .

**Comment by martinpitt** [comment]
Moving to draft until the release is done and the bodhi URLs filled in.

**Comment by martinpitt** [comment]
Release is done, https://bodhi.fedoraproject.org/updates/?packages=cockpit has the URLs.

**Pull request merged**